packages/mask/src/social-network/ui.ts

```diff
@@ -137,9 +145,16 @@ export async function activateSocialNetworkUIInner(ui_deferred: SocialNetworkUI.
                 },
                 personaSign: Services.Identity.signWithPersona,
                 walletSign: Services.Ethereum.personalSign,
+                privileged_silentSign: () => {
```

Comment:
It should not be a higher ordered function. Don't return a new function but call it directly.